sonic_installer/main.py

```diff
@@ -8,7 +8,9 @@
 import time
 import click
 import urllib
+import tempfile
 import subprocess
+import sonic_platform.chassis
```

Comment:
not all platform has this module implemented, so this will break all platforms that do not have this module defined. we need to check the module availability.

Reply:
Addressed in new commit...

Reply:
Thanks. We will need to do this until all vendors have transitioned to the new platform API.